docs/docs/docs/github.md

```diff
@@ -17,6 +17,7 @@ In order to allow *sbt-hood* to communicate with GitHub you'll need to set these
 * `repositoryName`: name of the repository where the plugin will post updates.
 * `pullRequestNumber`: pull request number where the plugin will post updates.
 * `targetUrl`: optional URL to the CI job, allowing GitHub to show a link to your CI job besides the mergeability status (i.e.: `TRAVIS_JOB_WEB_URL` under TravisCI).
+* `shouldBlockMerge`: if set to `true`, *sbt-hood* will ask GitHub to block the current PR in case a benchmark has produced errors. Defaults to `true`.
```

Comment:
sorry not familiar enough with sbt-hood. Is it that the status becomes red if they fail or if they are below an acceptable threshold?

Reply:
@BenFradet yup that's it! If at least one of the benchmarks fail (that is: they go below the expected threshold), the PR status is marked as unmergeable. Even though the status is marked with a tag "sbt-hood" and I think repos can be configured to ignore these tags selectively, after talking with @juanpedromoreno we decided it would be a good idea to offer this a separate setting (disabled by default). I wouldn't want sbt-hood to get in the way in projects that need some experimenting or multiple PRs to fix performance issues :).

Reply:
Thanks! :+1: